Missing conda packages from installation instructions

The build instructions fail for the conda environment that is created according to the installation instructions. On macOS, I got this error:

```
$ cmake -D CMAKE_INSTALL_PREFIX=$CONDA_PREFIX ..
-- Building xeus-calc v0..
CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  By not providing "Findxtl.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "xtl", but
  CMake did not find one.
...
```

I could fix that by following the alternative instructions in CONTRIBUTING.md:

```
conda create -n xeus-calc -c conda-forge -c defaults xtl nlohmann_json cppzmq xeus
```

--- a/README.md
+++ b/README.md
@@ -21,9 +21,7 @@ To ensure that the installation works, it is preferable to install `xeus` in a f
 The safest usage is to create an environment named `xeus-calc-env` within your miniconda installation
 
 ```bash
-conda create -n xeus-calc-env
-conda activate xeus-calc-env
-conda install -c conda-forge xeus
+conda create -n xeus-calc-env -c conda-forge -c defaults xtl nlohmann_json cppzmq xeus
 ```
 
 Once you have installed ` xeus ` ,the  ` xeus-calc ` project is quite easy to install, you have to clone the repository, and run the following command lines :

